elementtable.go

```diff
@@ -55,37 +56,49 @@ var table = elementTable{
+	ElementMinCache:               elementDef{[]byte{0x6D, 0xE7}, DataTypeUInt, false},
 	ElementTag:                    elementDef{[]byte{0x73, 0x73}, DataTypeMaster, false},
 	ElementSegmentFilename:        elementDef{[]byte{0x73, 0x84}, DataTypeString, false},
 	ElementSegmentUID:             elementDef{[]byte{0x73, 0xA4}, DataTypeBinary, false},
 	ElementTrackUID:               elementDef{[]byte{0x73, 0xC5}, DataTypeUInt, false},
 	ElementTitle:                  elementDef{[]byte{0x7B, 0xA9}, DataTypeString, false},
 	ElementTrackType:              elementDef{[]byte{0x83}, DataTypeUInt, false},
 	ElementCodecID:                elementDef{[]byte{0x86}, DataTypeString, false},
+	ElementFlagDefault:            elementDef{[]byte{0x88}, DataTypeUInt, false},
+	ElementFlagInterlaced:         elementDef{[]byte{0x9A}, DataTypeUInt, false},
+	ElementBlockDuration:          elementDef{[]byte{0x9B}, DataTypeUInt, false},
+	ElementFlagLacing:             elementDef{[]byte{0x9C}, DataTypeUInt, false},
 	ElementChannels:               elementDef{[]byte{0x9F}, DataTypeUInt, false},
-	ElementSimpleBlock:            elementDef{[]byte{0xA3}, DataTypeBlock, false},
 	ElementBlockGroup:             elementDef{[]byte{0xA0}, DataTypeMaster, false},
-	ElementBlockDuration:          elementDef{[]byte{0x9B}, DataTypeUInt, false},
 	ElementBlock:                  elementDef{[]byte{0xA1}, DataTypeBlock, false},
+	ElementSimpleBlock:            elementDef{[]byte{0xA3}, DataTypeBlock, false},
 	ElementPosition:               elementDef{[]byte{0xA7}, DataTypeUInt, false},
+	ElementCodecDecodeAll:         elementDef{[]byte{0xAA}, DataTypeUInt, false},
 	ElementPrevSize:               elementDef{[]byte{0xAB}, DataTypeUInt, false},
 	ElementTrackEntry:             elementDef{[]byte{0xAE}, DataTypeMaster, false},
 	ElementPixelWidth:             elementDef{[]byte{0xB0}, DataTypeUInt, false},
+	ElementCueDuration:            elementDef{[]byte{0xB2}, DataTypeUInt, false},
 	ElementCueTime:                elementDef{[]byte{0xB3}, DataTypeUInt, false},
 	ElementSamplingFrequency:      elementDef{[]byte{0xB5}, DataTypeFloat, false},
 	ElementCueTrackPositions:      elementDef{[]byte{0xB7}, DataTypeMaster, false},
+	ElementFlagEnabled:            elementDef{[]byte{0xB9}, DataTypeUInt, false},
 	ElementPixelHeight:            elementDef{[]byte{0xBA}, DataTypeUInt, false},
 	ElementCuePoint:               elementDef{[]byte{0xBB}, DataTypeMaster, false},
 	ElementTrackNumber:            elementDef{[]byte{0xD7}, DataTypeUInt, false},
 	ElementVideo:                  elementDef{[]byte{0xE0}, DataTypeMaster, false},
 	ElementAudio:                  elementDef{[]byte{0xE1}, DataTypeMaster, false},
 	ElementTimecode:               elementDef{[]byte{0xE7}, DataTypeUInt, false},
-	ElementVoid:                   elementDef{[]byte{0xEC}, DataTypeMaster, false},
+	ElementVoid:                   elementDef{[]byte{0xEC}, DataTypeBinary, false},
```

Comment:
The void element is binary: [spec](https://github.com/ietf-wg-cellar/ebml-specification/blob/a61dd6f88a7701dc27c27a5a79b3c187fd2159bb/specification.markdown#void-element)